VMWare/vmware-esxi/tests/AUTH_SSH_process_closed.json

```diff
@@ -0,0 +1,37 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "vmware-esxi",
+        "dialect_uuid": "2b13307b-7439-4973-900a-2b58303cac90"
+      }
+    },
+    "message": "   error hostd[2099655] [Originator@6876 sub=Vimsvc.ha-eventmgr] Event 974676 : SSH session was closed for 'root@1.2.3.4'."
+  },
+  "expected": {
+    "message": "   error hostd[2099655] [Originator@6876 sub=Vimsvc.ha-eventmgr] Event 974676 : SSH session was closed for 'root@1.2.3.4'.",
+    "observer": {
+      "vendor": "VMware",
+      "product": "ESXi"
+    },
+    "event": {
+      "kind": "event",
+      "category": [
+          "process",
+          "session"
+      ],
+      "type": [
+        "error"
```

Comment:
`error` for `event.type` seems only suitable for category `web` and `database` in the ECS specification.
I think a more suitable `event.type` will be `["end"]`, as it represent the end of the session and put the log level in `log.level`